packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -118,44 +118,50 @@ export function makeScreenshotEditorComponent({ h, imageBuffer, dialog }: Factor
       setConfirmCrop(false);
       const handleMouseMove = makeHandleMouseMove(corner);
       const handleMouseUp = (): void => {
-        croppingRef.current && croppingRef.current.removeEventListener('mousemove', handleMouseMove);
+        DOCUMENT.removeEventListener('mousemove', handleMouseMove);
         DOCUMENT.removeEventListener('mouseup', handleMouseUp);
         setConfirmCrop(true);
       };
 
       DOCUMENT.addEventListener('mouseup', handleMouseUp);
-      croppingRef.current && croppingRef.current.addEventListener('mousemove', handleMouseMove);
+      DOCUMENT.addEventListener('mousemove', handleMouseMove);
     }
 
     const makeHandleMouseMove = useCallback((corner: string) => {
       return function (e: MouseEvent) {
+        if (!croppingRef.current) {
+          return;
+        }
+        const cropCanvas = croppingRef.current;
+        const mouseX = e.clientX - cropCanvas.getBoundingClientRect().x;
+        const mouseY = e.clientY - cropCanvas.getBoundingClientRect().y;
```

Comment:
`getBoundingClientRect()` forces layout meaning it'll cause the browser to make a synchronous call. we want to avoid this as much as possible for browser perf. Let's make sure this is only called once